Web/Edubase.Web.UI/Areas/Establishments/Views/Establishment/Partials/TabGovernance.cshtml

```diff
@@ -32,7 +32,7 @@
                 {
                     using (Html.BeginRouteForm("EstablishmentGovernanceConfirmUpToDate", new {Model.Establishment.Urn, showBanner = true}, FormMethod.Post, new {@class = "gias-action-form" }))
                     {
-                    <button class="govuk-button" data-module="govuk-button" id="estab-data-confirm">Confirm governance information</button>
```

Comment:
Good catch with renaming this to refer explicitly to the establishment's governance data being confirmed! 👍